sdk_container/src/third_party/coreos-overlay/sys-apps/systemd/systemd-255.8.ebuild

```diff
@@ -258,7 +255,7 @@ src_unpack() {
 
 src_prepare() {
 	local PATCHES=(
-		"${FILESDIR}"/255-install-format-overflow.patch
+		"${FILESDIR}/systemd-test-process-util.patch"
```

Comment:
This patch `systemd-test-process-util.patch` is not per se required for Flatcar, maybe we can remove it too?

Reply:
Since this patch is pulled from upstream, I would suggest to keep this patch around to keep the diff as minimal as possible.

Reply:
I thought so too, just wanted to get another opinion.